ops-bedrock/docker-compose.yml

```diff
@@ -17,6 +17,7 @@ services:
       dockerfile: Dockerfile.l1
     ports:
       - "8545:8545"
+      - "8546:8546"
```

Comment:
Why export the WS ports?

Reply:
The rust challenger agent uses it

Reply:
^ We won't always depend on it, but for now, we use an event subscription with a fallback to a timer-based check on the `claimData` / `disputeGameList` array lengths.

Reply:
Ok. On the go side we've built a polling layer which will allow HTTP clients to create subscriptions, but I guess we don't have that on the rust side yet.